backend/internal/data/shutter_explorer.sql.go

```diff
@@ -229,6 +229,38 @@ func (q *Queries) QueryLatestTXsWhichArentIncluded(ctx context.Context, limit in
 	return items, nil
 }
 
+const queryTotalRegisteredValidators = `-- name: QueryTotalRegisteredValidators :many
+SELECT COUNT(DISTINCT validator_index) AS total_registered_validators
```

Comment:
I think this query is not quite correct. 

It would filter out the following case 

(Validator registers)
Validator deregisters
Validator registers again

Reply:
fixed